var/spack/repos/builtin/packages/qemu/package.py

```diff
@@ -0,0 +1,180 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+# ----------------------------------------------------------------------------
+# If you submit this package back to Spack as a pull request,
+# please first remove this boilerplate and all FIXME comments.
+#
+# This is a template package file for Spack.  We've put "FIXME"
+# next to all the things you'll want to change. Once you've handled
+# them, you can save this file and test your package like this:
+#
+#     spack install qemu
+#
+# You can edit this file again by typing:
+#
+#     spack edit qemu
+#
+# See the Spack documentation for more information on packaging.
+# ----------------------------------------------------------------------------
+
+from spack import *
+
+
+class Qemu(AutotoolsPackage):
+    """QEMU is a generic and open source machine emulator and virtualizer."""
+
+    # FIXME: Add a proper url for your package's homepage here.
+    homepage = "https://www.qemu.org"
+    url      = "https://download.qemu.org/qemu-4.1.1.tar.xz"
+
+    # FIXME: Add a list of GitHub accounts to
+    # notify when the package is updated.
+    # maintainers = ['github_user1', 'github_user2']
+
+    version('4.2.0-rc3',  sha256='5d5aece78673c758f648b3ae06ea9758ffe0345b7db07d92f3e5ce9bdcd55ed8')
```

Comment:
You don't actually need to add every single version if you don't want to. I usually only add the latest stable release, and I rarely add release candidates.